playbooks/setup_jenkins_slave.yml

```diff
@@ -116,6 +121,19 @@
           retries: 3
           delay: 2
 
+    - block:
+        - name: Create the workspace venv
+          pip:
+            requirements: "/opt/rpc_gating_requirements.txt"
+            virtualenv: "{{ lookup('env', 'WORKSPACE') }}/.venv }}"
```

Comment:
This playbook runs on a long-running-slave to prepare a single use slave. WORKSPACE refers to an absolute location on the long-running-slave and is not relevant on the single use slave. The location of the workspace on the single use slave is not known for sure until that node is used via a node() call.